models/Donation.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const CustomerSchema = new Schema({
+  id: Number,
+  name: String,
+  phone_number: String,
+  email: String,
+  created_at: Date
+})
+
+const DataSchema = new Schema({
```

Comment:
@Chiemezuo, let's rename this to `Transaction` schema

Reply:
On it! @edustipend-root